pkg/sqs/Symfony/SqsTransportFactory.php

```diff
@@ -34,6 +34,7 @@ public function addConfiguration(ArrayNodeDefinition $builder)
     {
         $builder
             ->children()
+                ->scalarNode('client')->defaultNull()->end()
```

Comment:
I would expect there to be something like the following in `createConnectionFactory()` below:

```
if (false == empty($config['client'])) {
    $config['client'] = new Reference($config['client']);
}
```